azure-functions/src/send_grid/message_builder.rs

```diff
@@ -831,12 +922,10 @@ impl MessageBuilder {
     }
 
     fn append_personalization(&mut self) {
```

Comment:
Feel free to rename this to `initialize_personalization` too.  I think your idea makes more sense.